Sources/MapboxMapsSwiftUI/MapboxView.swift

```diff
@@ -0,0 +1,87 @@
+@_exported import MapboxMaps
+import SwiftUI
+
+public extension CameraState {
+    /// Initializes CameraState with center and zoom.
+    init(center: CLLocationCoordinate2D, zoom: CGFloat) {
+        self.init(center: center, padding: .zero, zoom: zoom, bearing: 0, pitch: 0)
+    }
+}
+
+/// The action that is called when the map is loaded.
+public typealias MapLoadedAction = (MapboxMap) -> Void
+
+/// View displaying Mapbox Map in SwiftUI.
+@available(iOS 13.0, *)
+public struct MapboxView: UIViewRepresentable {
+    public typealias InitialOptionsProvider = () -> MapInitOptions
+
+    @Binding
+    var camera: CameraState
+    var cameraBounds: CameraBoundsOptions?
+
+    var annotations = [PointAnnotation]()
+    var _onMapLoaded: MapLoadedAction?
+    var styleURI = StyleURI.streets
+    var getstureOptions: GestureOptions = GestureOptions()
+
+    private let initialOptions: InitialOptionsProvider?
+
+    /// Creates an instance showing scpecisif region.
+    ///
+    /// - Parameters:
+    /// - camera: The camera state to display.
+    /// - initialOptions: A closure to provide initial map parameters. It gets called only once when `MapboxView` is created.
+    public init(camera: Binding<CameraState>, initialOptions: InitialOptionsProvider? = nil) {
```

Comment:
if there is `cameraOptions` in `MapInitOptions`, would it override the binding camera, or would it be ignored?

Reply:
The camera will override it. 
Ideally, I would exclude cameraOptions from `MapInitOptions` in SwiftUI in order to have only one way to set a camera, but it would require creating  alternative to MapInitOptions which in overcomplication at this moment imo.